controller/controller-worker/src/main/java/com/pinterest/rocksplicator/controller/WorkerConfig.java

```diff
@@ -35,10 +35,12 @@
   private static final String WORKER_POOL_SIZE_KEY = "worker_pool_size";
   private static final String DISPATCHER_POLL_INTERVAL_KEY = "dispatcher_poll_interval";
   private static final String ZK_PATH_KEY = "zk_path";
+  private static final String ZK_ENDPOINTS_KEY = "zk_endpoints";
   private static String HOST_NAME;
   private static PropertiesConfiguration configuration;
 
   private static final String DEFAULT_ZK_PATH = "/config/services/rocksdb/";
+  private static final String DEFAULT_ZK_ENDPOINTS = "observerzookeeper010:2181";
```

Comment:
nit: do we support a list of zk endpoint? i am not sure about curator, but for other zk clients, they support pass in endpoints separated by comma.

but given our very low load to zk, probably not a issue for now.